CHANGELOG.md

```diff
@@ -2,10 +2,17 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.15.0 - 2020-09-14
+### Added
+- PostgreSQL support to Attached Accounts
+- PostgreSQL support to API
```

Comment:
If we make the default version 10 (as discussed offline), that should be included in a `### Changed` section